[MIG] l10n_ar_currency_update/: Migration to 15.0

rafaelaxcelere opened the pull request.

Commit:
[MIG] l10n_ar_currency_update/: Migration to 15.0

Commit:
[MIG] l10n_ar_edi_ux: Migration to 15.0

roboadhoc commented:
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/odoo-argentina-ee/pull/61).

Commit:
[MIG] l10n_ar_account_tax_settlement: Migration to 15.0

Commit:
[MIG] account_tax_settlement: Migration to 15.0

Commit:
[MIG][l10n_ar_edi_ux] # TODO: Temporally commented to avoid account_payment_group addon

Commit:
[MIG][l10n_ar_edi_ux] Compatibility with Odoo 15 codes

-Some fields doesnt exist more
-Some fields change field name
-Some structure in views change
..

jjscarafia commented:
continue on https://github.com/ingadhoc/odoo-argentina-ee/pull/69/files
